src/jit/morph.cpp

```diff
@@ -4696,6 +4696,8 @@ GenTreePtr Compiler::fgMorphMultiregStructArg(GenTreePtr arg, fgArgTabEntryPtr f
     // We should still have a TYP_STRUCT
     assert(argValue->TypeGet() == TYP_STRUCT);
 
+    // TODO(pdg): below is where we'll need to create aggregate nodes.
+
     GenTreeArgList* newArg = nullptr;
```

Comment:
Better yet, this can be removed, as this has already been done. The TODO is leftover from my initial notes.